tests/app/test_event_handlers.py

```diff
@@ -1,6 +1,7 @@
 import uuid
 from unittest.mock import ANY
 
+from app import create_url
```

Comment:
In addition to changing the name, we'll need to add the redirect handler here as well:

```suggestion
from app import create_beta_url, redirect_notify_to_beta
```